crates/stackable-operator/src/patchinator/crd.rs

```diff
@@ -0,0 +1,20 @@
+use kube::api::DynamicObject;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::utils::crds::raw_object_list_schema;
+
+#[derive(Clone, Debug, Deserialize, JsonSchema, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ObjectOverrides {
+    /// A list of generic Kubernetes objects, which are merged onto the objects that the operator
+    /// creates.
```

Comment:
```suggestion
    /// A list of generic Kubernetes objects, which are merged with objects created by the operator.
```

Reply:
What about this?
```suggestion
    /// A list of generic Kubernetes objects, which are merged onto the objects created by the operator.
```
It makes it clear what is merged onto what